frontend/src/App.tsx

```diff
@@ -73,7 +76,7 @@ export default function App () {
                     toolsHide={true}
                     breadcrumbs={<BreadcrumbsProvider/>}
                 />
-                { window.env.SYSTEM_BANNER?.text && <SystemBanner position='BOTTOM' /> }
+                { applicationConfig.configuration.SystemBanner!.isEnabled && <SystemBanner position='BOTTOM' /> }
```

Comment:
Same comment as above. Remove the `!` if it is unnecessary, which it appears to be.